### ISSUE TITLE
devilutionx.mpq missing from the Mac OS bundel

### ISSUE THREAD

**Issue by DrGonzo564** [diasurgical/devilutionX#1554]
**Important information**
Mac OS 10.15.7 iMac intel / DevilutionX 1.2.1 / Downloaded a release

**Describe**
Editing the Enemy Health Bar from =0 to =1 is causing DevilutionX 1.2.1 to crash when attempting to enter a game it will crash right after game speed selection screen with Error "Failed to load U1 resources Is devilutionx.mpq accessible and up to date?"

**To Reproduce**
1. Go to ~/Library/Application Support/diasurgical/devilution'
2. Open 'diablo.ini'
3. Change "Enemy Health Bar=0" to "Enemy Health Bar=1"
4. Save file
5. load DevilutionX 
6. Select Multiplayer > Loopback > Select Hero > Difficulty > Speed > CRASH :( 

**Expected behavior**
I wanted to see Enemy Health Bars.
<img width="1557" alt="Screen Shot 2021-04-15 at 9 24 40 PM" 

**Screenshots**
<img width="1557" alt="Screen Shot 2021-04-15 at 9 24 40 PM" src="https://user-images.githubusercontent.com/82344993/114972574-9aa5a600-9e33-11eb-9e59-6d2c52847ebf.png">

**Additional context**
I have been able to edit other .ini settings (auto equip weapons, ex bar) without problems but turning on Enemy Health Bar seems to cause is to crash every time for me.

**Comment by AJenbo** [comment]
This isn't a crash, it's an error saying you are missing devilutionx.mpq

You can get it here: https://github.com/diasurgical/devilutionX/blob/master/Packaging/resources/devilutionx.mpq

Oviously it would be ideal if the data was embedded in the bunde, so lets keep the issue open to track that.

**Comment by DrGonzo564** [comment]
Ahh my bad didn't know this. Thanks!

**Comment by AJenbo** [comment]
> it would be ideal if the data was embedded in the bundle, so let's keep the issue open to track that.

**Comment by glebm** [comment]
We can probably include the assets directly into the app container.

**Comment by AJenbo** [comment]
yeah, gave it a quick shot, but there is some tweak we need to do to the code first. It's odd since that is how we used to load the ttf file.

**Comment by glebm** [comment]
The assets are now included into the app container.
